Lock/AuthPresenter.swift

```diff
@@ -48,7 +48,7 @@ class AuthPresenter: Presentable, Loggable {
 
     private func newView(withInsets insets: UIEdgeInsets, mode: AuthCollectionView.Mode) -> AuthCollectionView {
         let view = AuthCollectionView(connections: self.connections, mode: mode, insets: insets, customStyle: self.customStyle) { name in
-            self.interactor.login(name, loginHint: nil) { error in
+            self.interactor.start(name, loginHint: nil, screenHint: nil) { error in
```

Comment:
Can these default to `nil` in the method signature instead? otherwise, every introduced param will make you add it here with nil as value. See this diff to get the picture:
![image](https://user-images.githubusercontent.com/3900123/91223859-a2bfa280-e6f7-11ea-8c98-19e9ae7eb76e.png)



For connection, if possible, I'd use the named param. Something like this:
```suggestion
            self.interactor.start(connection: name) { error in
```